src/Avalonia.Controls/PlacementMode.cs

```diff
@@ -13,28 +13,74 @@ public enum PlacementMode
         Pointer,
 
         /// <summary>
-        /// The popup is placed at the bottom left of its target.
+        /// Preferred location is below the target element.
         /// </summary>
         Bottom,
 
         /// <summary>
-        /// The popup is placed at the top right of its target.
+        /// Preferred location is to the right of the target element.
         /// </summary>
         Right,
         
         /// <summary>
-        /// The popup is placed at the top left of its target.
+        /// Preferred location is to the left of the target element.
         /// </summary>
         Left,
         
         /// <summary>
-        /// The popup is placed at the top left of its target.
+        /// Preferred location is above the target element.
         /// </summary>
         Top,
         
         /// <summary>
-        /// The popup is placed according to anchor and gravity rules
+        /// Preferred location is centered over the target element.
         /// </summary>
-        AnchorAndGravity
+        Center,
+
+        /// <summary>
+        /// The popup is placed according to <see cref="Popup.PlacementAnchor"/> and <see cref="Popup.PlacementGravity"/> rules.
+        /// </summary>
+        AnchorAndGravity,
```

Comment:
@amwx I remember previously FlyoutPlacementMode was implemented with keeping same enum values as UWP had. This PR breaks that, as I would prefer to keep popup PlacementMode values stable instead. Please let me know, if there could be a problem with that.

Reply:
That should be fine. I think the values were only there from copying the enum anyway